packages/core/src/Accordion/Accordion.stories.tsx

```diff
@@ -100,7 +100,10 @@ export const Ladder = {
     />
   ),
 }
-export const Card = { render: (args) => <Accordion {...args} items={items} variation='card' /> }
+
+export const Card = {
+  render: (args) => <Accordion {...args} items={items} variation='card' itemsState={['item-1', 'item-2']} />,
+}
```

Comment:
Could you make a new story for when the `itemState` prop is present?

Reply:
Can you clarify on that?

Reply:
It would be helpful to keep the existing `Card` story, but then create an additional story that uses the `itemsState` prop